### PR TITLE
Button Component

### PR DESCRIPTION
**Pull request by kalyandindu**
Added button component, It includes Primary , Outline, Link, button with icon etc.

image

### CODE REVIEW

**Review by nishasen — APPROVED** [review]
You have used quite a lot variables, code looks clean to me.

File: Docs/Button/button.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="/CSS/main.css">
+  <title>Avatar</title>
```
Comment (therajatg) [inline Docs/Button/button.html:9]: change to button

**Review by therajatg — COMMENTED** [review]
Everything else looks good

File: Docs/Button/button.html (hunk 1 above)
Comment (kalyandindu) [reply Docs/Button/button.html:9]: Thanks for the observation, Sure will change

File: CSS/Components/button.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  color: var(--white);
+}
+
+.btn-wrapper {
+  display: flex;
+  align-items: center;
+  gap: var(--sp-xsm);
+}
+
+/* To make a button roundable*/
```
Comment (Sameony) [inline CSS/Components/button.css:65]: such comments are unnecessary as the class names are descriptive enough

File: Docs/Button/button.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+      <h3 class="content_h3">Fixed Button:</h3>
+      <div class="btn-container d-flex">
+        <p class="btn-container_p"> Include btn-float class to make a button floatable</p>
+        <!-- <button class="btn btn-outline rounded btn-float flex-center">
+         <i class="fa-solid fa-moon"></i>
+        </button>
+        <button class="btn btn-outline rounded btn-float flex-center">
+          <i class="fa-solid fa-circle-plus"></i>
+         </button> -->
```
Comment (Sameony) [inline Docs/Button/button.html:78]: remove unnecessary comments or code blocks before merging

**Review by Sameony — COMMENTED** [review]
do follow up on the adjustments and changes recommended. 
Also, avoid using ambiguous/same commits multiple times

**Pull request merged**